components/aws/sagemaker/deploy/src/deploy.py

```diff
@@ -53,15 +54,16 @@ def main(argv=None):
 
   logging.getLogger().setLevel(logging.INFO)
   client = _utils.get_sagemaker_client(args.region, args.endpoint_url, assume_role_arn=args.assume_role)
+  endpoint_config_name = _utils.get_current_endpoint_config(client, vars(args))
```

Comment:
should only be called if endpoint name has been passed
also lets pass only endpoint_name rather than all arguments

Reply:
Addressed comments